packages/keyring/src/testing.ts

```diff
@@ -77,42 +77,42 @@ export const PAIRSETHEREUM: PairDef[] = [
     name: 'Alith',
     publicKey: hexToU8a('0x02509540919faacf9ab52146c9aa40db68172d83777250b28e4679176e49ccdd9f'),
     secretKey: hexToU8a('0x5fb92d6e98884f76de468fa3f6278f8807c48bebc13595d45af5bdc4da702133'), // nosemgrep
-    seed: "m/44'/40'/0'/0",
+    seed: "m/44'/60'/0'/0",
     type: 'ethereum'
   },
   {
     name: 'Baltathar',
     publicKey: hexToU8a('0x033bc19e36ff1673910575b6727a974a9abd80c9a875d41ab3e2648dbfb9e4b518'),
     secretKey: hexToU8a('0x8075991ce870b93a8870eca0c0f91913d12f47948ca0fd25b49c6fa7cdbeee8b'), // nosemgrep
-    seed: "m/44'/40'/0'/0",
+    seed: "m/44'/60'/0'/0",
```

Comment:
This won't break where it is already used?

Reply:
actually the seed here is just a filler because the private key has already been derived so let me remove it

Reply:
Ahh, yes, you are right.  It was basically there as a hint of how to get to it.